lib/kinetic_cafe/error_dsl.rb

```diff
@@ -141,8 +141,7 @@ def define_error(options)
       status ||= defined?(Rack::Utils) && :bad_request || 400
       status.freeze
 
-      severity ||= :error
-      severity.freeze
+      severity ||= :warn
```

Comment:
since it is a symbol, we don't have to freeze it, right? symbols are not strings and are already immutable, right?